recipe/meta.yaml

```diff
@@ -31,13 +33,14 @@ test:
     - {{ compiler('c') }}
   files:
     - test.cc
+    - test.sh
   commands:
     - test -f $PREFIX/lib/libdouble-conversion.a      # [unix]
     - test -f $PREFIX/lib/libdouble-conversion.so     # [linux]
     - test -f $PREFIX/lib/libdouble-conversion.dylib  # [osx]
```

Comment:
The linter complains about test commands and test files. If the file run_test.{sh,bat} exist, then move commands to the test file. Otherwise, test commands won't be executed. If you check logs, there is no execution of the test.commands, but only run_test.sh.

The second. Does the file `test.sh` exist in the recipe folder? See https://github.com/AnacondaRecipes/double-conversion-feedstock/tree/PKG-7961_double-conversion_3.3.1/recipe

Looks like the feedstock doesn't have README.md and a license? https://github.com/AnacondaRecipes/double-conversion-feedstock/tree/PKG-7961_double-conversion_3.3.1